### ISSUE TITLE
gaps left all set 280px errors

### ISSUE THREAD

**Issue by RossComputerGuy** [Airblader/i3#274]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior
```
[{"success":false,"parse_error":true,"error":"Expected one of these tokens: 'inner', 'outer'","input":"gaps left all set 280px","errorposition":"     ^^^^^^^^^^^^^^^^^^"}]
```

## Expected Behavior
Setting the left gap to 280px

## Reproduction Instructions
run `i3 gaps left all set 280px` in a terminal.

## Environment

Output of `i3 --moreversion 2>&-`:
<pre>
Binary i3 version:  4.16 (2018-11-04) © 2009 Michael Stapelberg and contributors
Running i3 version: 4.16 (2018-11-04) (pid 20364) abort…)
Loaded i3 config: /home/spaceboyross/.config/i3/config (Last modified: Sat 12 Jan 2019 10:30:05 PM PST, 275 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3
</pre>

<pre>
gaps left all set 280
</pre>

<pre>
Logfile URL:
</pre>

<pre>
- Linux Distribution & Version: Arch Linux
- Are you using a compositor (e.g., xcompmgr or compton): compton
</pre>

**Comment by Airblader** [comment]
4.16 doesn't include this feature yet, it was merged afterwards.

**Comment by RossComputerGuy** [comment]
Then the readme is misleading.

**Comment by Airblader** [comment]
Not if you look at the README on the correct branch, which is "gaps". "gaps-next" is the development branch. This is also explained in the README.

**Comment by RossComputerGuy** [comment]
I'm looking on the main one.

**Comment by Airblader** [comment]
The default branch is the development branch. I can't possibly know which version you get from your distribution when you install a package. When you install the latest HEAD, that's what the README definitely applies to.

**Comment by RossComputerGuy** [comment]
It may be misleading to have the main branch be the dev branch because people who look for the docs will see the usage and not notice it's the dev branch when they're running the normal release.

**Comment by Airblader** [comment]
I'm not sure what I can do against that. I don't want to change the default branch, because GitHub is where development happens and I want to make sure PRs etc go against the development branch by default. Not to mention that's how it's setup at i3.

I can't put a hint into the README since the next release would merge that hint to the gaps branch, too. The best idea would be to keep the docs on a website and not visibly in the README in the repository, but that's quite a bit of work. 

A compromise would be to add a note in the README that these options are only available with 4.17+. I can pretty much guarantee that no one will read this, but it's about the only thing coming to mind for me.
